### ISSUE TITLE
Insufficient color contrast ratio

### ISSUE THREAD

**Issue by MiChAeLoKGB** [ChrisRM/material-theme-jetbrains#696]
Colors used in Material Dark theme have insufficient contrast ratio (font color vs background color).

The [minimal contrast ratio](https://www.w3.org/TR/WCAG/#contrast-minimum) should be 4.5:1, but the color of files in sidebar has [contrast ratio of 1.7:1](http://leaverou.github.io/contrast-ratio/#rgb%2866%2C%2066%2C%2066%29-on-rgb%2826%2C%2026%2C%2026%29)

Also, excluded folders are even worse, with their background color and [contrast ratio of 1:5:1](http://leaverou.github.io/contrast-ratio/#rgb%2897%2C%2097%2C%2097%29-on-rgb%2873%2C%2071%2C%2063%29)...

![phpstorm64_2018-03-26_10-08-01](https://user-images.githubusercontent.com/6197503/37894420-8297a88a-30de-11e8-98b8-ef0a00ab49fe.png)

I mean, its readable for me, but somebody with eyesight issues, would probably not be able to see anything...
You could try [No Coffee for Chrome](https://chrome.google.com/webstore/detail/nocoffee/jjeeggmbnhckmgdhmgdckeigabjfbddl) to simulate how such person would see your design.

### Environment

PhpStorm 2018.1 EAP
Build #PS-181.4203.513, built on March 21, 2018
JRE: 1.8.0_152-release-1136-b20 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

Material Theme UI
Version: 2.1.3

**Comment by mallowigi** [comment]
I’m aware of the text color in Darker. I’m planning to increase a bit this, but it will probably not be the ideal contrast ratio that you seek

**Comment by MiChAeLoKGB** [comment]
I dont really need it to be 4.5:1, but at least 3:1 would be nice.

Compare those two for example:
http://leaverou.github.io/contrast-ratio/#rgb%2866%2C%2066%2C%2066%29-on-rgb%2826%2C%2026%2C%2026%29

http://leaverou.github.io/contrast-ratio/#rgb%28100%2C%20100%2C%20100%29-on-rgb%2825%2C%2025%2C%2025%29

The second one is much better.

**Comment by Gaulomatic** [comment]
Does anyone know which file to edit to change that manually in the meantime? I simply cant find it.

**Comment by mallowigi** [comment]
I increased the luminance in the 2.1.5.3 version. Take a look there
